src/verilog/verilog_expr.h

```diff
@@ -784,51 +784,73 @@ inline verilog_caset &to_verilog_case(exprt &expr)
 class verilog_ift:public verilog_statementt
 {
 public:
-  verilog_ift():verilog_statementt(ID_if)
+  verilog_ift(exprt __cond, verilog_statementt __then_case)
+    : verilog_statementt(ID_if)
   {
-    operands().resize(3);
+    add_to_operands(std::move(__cond), std::move(__then_case));
   }
 
-  exprt &condition()
+  verilog_ift(
+    exprt __cond,
+    verilog_statementt __then_case,
+    verilog_statementt __else_case)
+    : verilog_statementt(ID_if)
+  {
+    add_to_operands(
+      std::move(__cond), std::move(__then_case), std::move(__else_case));
+  }
```

Comment:
Just a thought: would it be simpler to always use 3 operands and, when necessary, make that third operand `nil`?

Reply:
Possibly; but a goal was to match `code_ifthenelset`, which would then also have to change.